packages/aserto/src/aserto/client/directory/aio/__init__.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional, Tuple
+
+import grpc
+from aserto.directory.common.v2 import ObjectTypeIdentifier, PaginationRequest
+from aserto.directory.exporter.v2 import ExporterStub
+from aserto.directory.importer.v2 import ImporterStub
+from aserto.directory.reader.v2 import GetObjectsRequest, GetObjectsResponse, ReaderStub
+from aserto.directory.writer.v2 import WriterStub
+
+
+class DirectoryAsync:
+    def __init__(self):
+        pass
+
+    @classmethod
+    async def create(cls, *, address: str, api_key: str, tenant_id: str, ca_cert: str):
```

Comment:
I think this is a great solution! A few suggestions to make the sync and async versions more interoperable.
1. Rename `DirectoryAsync` to `Directory` just like the sync version. This will allow users to switch from one implementation to the other by changing the import.
2. Change `__init__` to take `channel, api_key, and tenant_id` as parameters and have `create` call it to return the new directory object (i.e. `return Directory(channel, api_key, tenant_id)`)
3. Add a synchronous `create` class method to the sync version of Directory too.